requirements.txt

```diff
@@ -1,8 +1,8 @@
-numpy==1.16.2
-scipy==1.1.0
-torch==1.1.0
-torchvision==0.2.2
-tqdm==4.19.9
-scikit-learn==0.20.3
-matplotlib==2.1.0
-visdom==0.1.8.5
\ No newline at end of file
+numpy==1.20.1
+scipy==1.6.1
+torch==1.8.0
+torchvision==0.9.0
+tqdm==4.59.0
+scikit-learn==0.24.1
+matplotlib==3.3.4
+visdom==0.1.8.9
```

Comment:
Dont want to mess with the requirements file. There's also the old requirements which are getting added to the repo.